src/Jenssegers/Mongodb/Auth/User.php

```diff
@@ -15,5 +16,5 @@ class User extends Model implements
     AuthorizableContract,
     CanResetPasswordContract
 {
-    use Authenticatable, Authorizable, CanResetPassword;
+    use Authenticatable, Authorizable, CanResetPassword, MustVerifyEmail;
```

Comment:
Check that we extends https://github.com/laravel/framework/blob/6.x/src/Illuminate/Foundation/Auth/User.php and remove interfaces, please

Reply:
It is hard because we should extends inheritor from https://github.com/jenssegers/laravel-mongodb/blob/master/src/Jenssegers/Mongodb/Eloquent/Model.php